### PR TITLE
The Apollo Team, Luminous Scans, Alpha Scans: Added Websites Support

### LINKED ISSUE #577: [website module request] luminousscans
**Website Information**
* **Name:**  LuminousScans
* **Root URL:** https://www.luminousscans.com
* **Directory list URL:** https://www.luminousscans.com/series/

* **Example manga info URL:** https://www.luminousscans.com/series/my-office-noonas-story/
* **Example chapter URL:** https://www.luminousscans.com/my-office-noonas-story-prologue/

**Languages**
English

### LINKED ISSUE #601: [website module request] luminousscans.com
Website Information

    Name: Luminousscans
    Root URL: https://www.luminousscans.com/
    Directory list URL: https://luminousscans.com/series/
    Example manga info URL: https://www.luminousscans.com/series/the-chronicles-of-heavenly-demon/
    Example chapter URL: https://www.luminousscans.com/volcanic-age-former-life-in-mount-hwa-chapter-176/
    Languages: English

### LINKED ISSUE #653: [website module request] The Apollo Team
**Website Information**
* **Name:** The Apollo Team
* **Root URL:** https://theapollo.team/
* **Directory list URL:** https://theapollo.team/manga/

* **Example manga info URL:** https://theapollo.team/manga/warring-states/
* **Example chapter URL:** https://theapollo.team/warring-states-chapter-22/

**Languages**
English

**Additional details**
Seem to use the same template as Luminous Scans(#691)

### LINKED ISSUE #691: [website module request] Luminous Scans
**Website Information**
* **Name:** Luminous Scans
* **Root URL:** https://www.luminousscans.com/
* **Directory list URL:** https://www.luminousscans.com/series/

* **Example manga info URL:** https://www.luminousscans.com/series/legend-of-the-northern-blade/
* **Example chapter URL:** https://www.luminousscans.com/legend-of-the-northern-blade-chapter-98/

**Languages**
English

**Additional details**
Seem to use the same template as The Apollo Team (#653)

### LINKED ISSUE #693: [website module request] Alpha Scans
**Website Information**
* **Name:** Alpha Scans
* **Root URL:** https://alpha-scans.org/
* **Directory list URL:** https://alpha-scans.org/manga/

* **Example manga info URL:** https://alpha-scans.org/manga/wind-sword/
* **Example chapter URL:** https://alpha-scans.org/wind-sword-chapter-91/

**Languages**
English

### PR DESCRIPTION
**Pull request by Trollam**
Add support for:
- The Apollo Team (resolves #653)
- Luminous Scans (resolves #691, resolves #601, resolves #577)
- Alpha Scans (resolves #693)

Maybe can support (with some tweaks):
- Olympus Scanlation (#682)

### CODE REVIEW

**Comment by Slasar41** [comment]
I think you can just put the websites to MangaShiro.lua instead of creating another module.

File: lua/modules/MangaShiro.lua [hunk 1]
```diff
@@ -294,7 +293,11 @@ function GetNameAndLink()
 			['41294a121062494489adfa601c442ef8'] = '/manga-list/?list', -- LegionAsia
 			['b53534f8443e420ea088594c53a3ff39'] = '/manhwa-list/', -- Manhwaland
 			['ff17b64aa945403dae45706753235872'] = '/latest-update/?list', -- KomikNesia
-			['5474e31b24ab4908a5258176d1f24f67'] = '/komik/list-mode/' -- ManhwaTaro
+			['5474e31b24ab4908a5258176d1f24f67'] = '/komik/list-mode/', -- ManhwaTaro
+			['f794803973af4e5daab21683d4de873a'] = '/series/list-mode/', -- LuminousScans
+			['86588503fd9e4277802c998cbccbc983'] = '/manga/list-mode/', -- AlphaScans
+			['275b85bdaafb47fdbc40f51d2bea99e8'] = '/manga/list-mode/' -- TheApolloTeam
```
Comment (Slasar41) [inline lua/modules/MangaShiro.lua]: You can remove this as the main dirurl is already there at line 302.

Comment (Trollam) [reply lua/modules/MangaShiro.lua]: I did not see that, done.

**Review by Slasar41 — APPROVED** [review]

**Comment by Slasar41** [comment]
Thanks.

**Pull request merged**